src/main/java/com/github/romankh3/image/comparison/ImageComparison.java

```diff
@@ -86,6 +86,17 @@ public void setThreshold(int threshold) {
      * @return the result of the drawing.
      */
     public BufferedImage compareImages() throws IOException {
+        return compareImages(1);
+    }
+
+    /**
+     * Draw rectangles which cover the regions of the difference pixels.
+     *
+     * @param lineWidth the width
+     * @return the result of the drawing.
+     * @throws IOException
+     */
+    public BufferedImage compareImages(int lineWidth) throws IOException {
```

Comment:
please, make the name of the variable more self-describing. For instance `rectangleLineWidth`.